import/index.template.html

```diff
@@ -543,56 +612,33 @@
           // In the case of import, this public key is not used because the client encrypts messages
           // to the server's public key.
           TKHQ.sendMessageUp("PUBLIC_KEY_READY", "");
-          // TODO: find a way to filter messages and ensure they're coming from the parent window?
-          // We do not want to arbitrarily receive messages from all origins.
+
```

Comment:
In other templates you moved this function to _after_ the call to `window.addEventListener`. I think that makes sense to do it here as well for consistency?

Reply:
Yes, agreed. I'll move that for consistency.

Reply:
Updated.